mergin/client.py

```diff
@@ -562,7 +563,21 @@ def __init__(self, url, auth_token=None, login=None, password=None, plugin_versi
                 "username": token_data["username"]
             }
 
-        self.opener = urllib.request.build_opener()
+        # fix for wrong macos installation of python certificates,
+        # see https://github.com/lutraconsulting/qgis-mergin-plugin/issues/70
+        # remove when https://github.com/qgis/QGIS-Mac-Packager/issues/32
+        # is fixed.
+        default_cafile = ssl.get_default_verify_paths().openssl_cafile
+        if os.path.exists(default_cafile):
```

Comment:
well, this gives me False on both Ubuntu (19.10) and Windows 10, nevertheless with shipped cert it seems to be working

Reply:
what path is in `ssl.get_default_verify_paths().openssl_cafile` ?

Reply:
alternatively if you want, you can add `platform.system() == 'Darwin'` or so